packages/fetchai/skills/tac_control_contract/behaviours.py

```diff
@@ -282,17 +282,11 @@ def _cancel_tac(self):
     def _create_items(self) -> TransactionMessage:
```

Comment:
How are you now distinguishing between the different tokens ("game currency" and rest)?